src/can.rs

```diff
@@ -1,657 +1,94 @@
-//! Controller Area Network (Requires feature `can`)
+//! Controller Area Network
 //!
 //! CAN is currently not enabled by default, and
 //! can be enabled by the `can` feature.
 //!
-//! It is a implementation of the [`embedded_hal_can`][can] traits.
+//! It is a implementation of the [`bxcan`][can] traits.
 //!
-//! [can]: embedded_hal_can
+//! [can]: bxcan
 //!
 //! A usage example of the can peripheral can be found at [examples/can.rs]
 //!
 //! [examples/can.rs]: https://github.com/stm32-rs/stm32f3xx-hal/blob/v0.6.1/examples/can.rs
 
-pub use embedded_hal_can::{self, Filter, Frame, Id, Receiver, Transmitter};
+use crate::gpio::{gpioa, gpiob};
+use crate::gpio::{PushPull, AF7, AF9};
 
-use crate::gpio::gpioa;
-use crate::gpio::{PushPull, AF9};
 use crate::pac;
 use crate::rcc::APB1;
-use nb::{self, Error};
+use bxcan::RegisterBlock;
```

Comment:
```suggestion
use bxcan::RegisterBlock;
pub use bxcan
```

Hm. I'm thinking about reexporting the `bxcan` crate, so one could use `bxcan` with this HAL without explicitly adding `bxcan` to the Cargo.toml itself. The suggestion is just a placeholder. I'm not sure how do to it the most clean way.